hydromt/data_adapter.py

```diff
@@ -711,6 +721,8 @@ def __init__(
         self.unit_add = unit_add
         # meta data
         self.meta = {k: v for k, v in meta.items() if v is not None}
+        # placeholders
+        self.placeholders = {k: v for k, v in placeholders.items() if v is not None}
```

Comment:
@laurenebouaziz: Is there a specific reason why you added this line? I think it can be removed.

Reply:
At some point the placeholders were added to *kwargs and this gave an error in open_mfdataset. So I made a few changes in the code to fix this, but I think you are right, this one is not needed. I will remove this line.